package.yml

```
@@ -31,3 +31,6 @@ requires:
 system_plugins:
     - manager
     - email
+
+installer_ignore:
+    - .tools/vendor
```

```suggestion
    - .tools
```
Ich habe nochmal einen Blick drauf geworfen und man könnte auch den kompletten `.tools` Ordner excluden